service/src/main/java/uk/gov/hmcts/reform/fpl/utils/PeopleInCaseHelper.java

```diff
@@ -28,19 +29,29 @@ public static String getFirstApplicantName(List<Element<Applicant>> applicants)
     }
 
     public static String getFirstRespondentLastName(List<Element<Respondent>> respondents) {
-        return ElementUtils.unwrapElements(respondents).stream()
-            .filter(Objects::nonNull)
-            .findFirst()
-            .map(Respondent::getParty)
+        return getFirstRespondent(respondents)
             .map(RespondentParty::getLastName)
             .orElse("");
     }
 
+    public static String getFirstRespondentFullName(List<Element<Respondent>> respondents) {
+        return getFirstRespondent(respondents)
+            .map(RespondentParty::getFullName)
+            .orElse("");
+    }
+
     //To be deleted when print and post is available to use as part of FPLA-1287
     public static List<String> formatRepresentativesForPostNotification(List<Representative> representatives) {
         return representatives.stream()
             .map(representative -> String.format("%s%n%s", representative.getFullName(),
                 representative.getAddress().getAddressAsString(", ")))
             .collect(toList());
     }
+
+    private static Optional<RespondentParty> getFirstRespondent(List<Element<Respondent>> respondents) {
```

Comment:
This is just getting the party part of it, not the whole respondent, name is a bit misleading